// Copyright 2019-2020 Chris Cummins <chrisc.101@gmail.com>.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto2";
import "libcecl.proto";

// Recording of a single benchmark execution.
message BenchmarkRun {
  required string hostname = 1;
  required string device_name = 2;
  required string benchmark_suite = 3;
  required string benchmark_name = 4;
  required string dataset_name = 5;
  required LibceclExecutableRun run = 6;
}
